test/genericschememulticall.js

```diff
@@ -445,6 +526,23 @@ contract('GenericSchemeMultiCall', function(accounts) {
        }
     });
 
+    it("execute proposeVote with multiple calls with votingMachine without whitelisted spender", async function() {
+      var actionMock =await ActionMock.new();
+      var standardTokenMock = await ERC20Mock.new(accounts[0],1000);
+      var testSetup = await setup(accounts,[actionMock.address],0,true,standardTokenMock.address);
+      var encodedTokenApproval= await createCallToTokenApproval(standardTokenMock, accounts[3], 1000);
+      var callData1 = await createCallToActionMock(testSetup.org.avatar.address,actionMock);
+      try {
+         await testSetup.genericSchemeMultiCall.proposeCalls(
+           [actionMock.address],
```

Comment:
this will revert due to wrong array length and not due to the spender not white listed